# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../luaexpat/makefile.patch
# Copyright (C) 2006 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./makefile.orig	2005-06-27 19:04:27.000000000 +0200
+++ ./makefile	2006-10-09 14:13:41.000000000 +0200
@@ -7,11 +7,8 @@
 
 lib: src/$(LIBNAME)
 
-src/$(LIBNAME) : src/lxplib.o $(COMPAT_DIR)/compat-5.1.o
-	export MACOSX_DEPLOYMENT_TARGET="10.3"; $(CC) -o src/$(LIBNAME) $(LIB_OPTION) src/lxplib.o $(COMPAT_DIR)/compat-5.1.o -lexpat
-
-$(COMPAT_DIR)/compat-5.1.o: $(COMPAT_DIR)/compat-5.1.c
-	$(CC) -c $(CFLAGS) -o $@ $(COMPAT_DIR)/compat-5.1.c
+src/$(LIBNAME) : src/lxplib.o
+	export MACOSX_DEPLOYMENT_TARGET="10.3"; $(CC) -o src/$(LIBNAME) $(LIB_OPTION) src/lxplib.o -lexpat
 
 install:
 	mkdir -p $(LUA_LIBDIR)
@@ -21,6 +18,6 @@
 	cp src/$T/lom.lua $(LUA_DIR)/$T
 
 clean:
-	rm -f src/$(LIBNAME) src/lxplib.o $(COMPAT_DIR)/compat-5.1.o
+	rm -f src/$(LIBNAME) src/lxplib.o
 
 # $Id: makefile,v 1.32 2005/06/27 17:04:27 tomas Exp $
